debug/db.py

```diff
@@ -239,18 +239,77 @@ def get_torrent(self, info_hash):
         Returns:
             Torrent: see torrent.py for more information
         """
-        pass
+        
+        """
+        info_hash BYTEA PRIMARY KEY,
+        name TEXT,
+        comment TEXT,
+        created_by TEXT,
+        creation_time TIMESTAMP,
+        piece_length INT,
+        pieces BYTEA
+        """
+        
+        cursor = self._connection.cursor()
+        try:
+            cursor.execute( ("SELECT * FROM torrents WHERE info_hash = 'asdfasdf'") )
+            tup = cursor.fetchone()
+        except psycopg2.ProgrammingError as e:
+            print(e)
+            return None
+        cursor.close()
+        
+        info_hash = bytes(tup[0])
+        name = tup[1]
+        comment = tup[2]
+        created_by = tup[3]
+        creation_time = tup[4]
+        piece_length = tup[5]
+        pieces = bytes(tup[6])
+
+        torrent = {}
```

Comment:
You are missing the 'announcer-lisr' key, another call will need made to the announcers table to get this information